tests/python_tests/conftest.py

```diff
@@ -17,6 +17,20 @@
 from helpers.perf import delete_reports
 
 
+def init_llk_home():
+    if "LLK_HOME" not in os.environ:
+        helpers = os.path.abspath(os.path.dirname(__file__))
+
+        os.environ["LLK_HOME"] = os.path.abspath(
+            os.path.join(
+                helpers,
+                "..",
+                "..",
+                "..",
+            )
+        )
```

Comment:
```suggestion
def init_llk_home():
   if "LLK_HOME" not in os.environ:
    try:
        helpers = Path(__file__).resolve().parent
        llk_home = helpers.parents[2]  # Up 3 levels
        os.environ["LLK_HOME"] = str(llk_home)
    except Exception as e:
        raise RuntimeError("Failed to set LLK_HOME from script location.") from e
```
Could use pathlib.Path for clearer + reliable path handling. Don't need to hardcode "..",  explicitly navigates directory levels.